internal/storage/migrations/0-init-schema.sql

```diff
@@ -377,4 +377,25 @@ create table if not exists events_sent (
 alter table events_sent
     add constraint events_sent_connector_id_fk foreign key (connector_id)
     references connectors (id)
+    on delete cascade;
+
+-- tasks
+create table if not exists tasks (
+    -- Mandatory fields
+    id varchar not null,
+    connector_id varchar not null,
+    status text not null,
+    created_at timestamp without time zone not null,
+    updated_at timestamp without time zone not null,
+
+    -- Optional fields
+    created_object_id varchar,
```

Comment:
In the case of fetch payments / fetch accounts / etc workflows a task has a 1 to many relationship with the objects it creates, right? So how will this work?

Reply:
This table will only be used for the creation of objects in the background I was thinking.

Reply:
ok perhaps I was confused by the new TaskTypes you added